src/main/resources/config/application.yaml

```diff
@@ -6,7 +6,7 @@ spring:
 
   cloud:
     function:
-      definition: consumeSaResult;consumeSaStopped;consumeSaFailed;consumeDsResult;consumeDsStopped;consumeDsFailed;consumeBuildResult;consumeBuildStopped;consumeBuildFailed;consumeCaseImportSucceeded;consumeCaseImportFailed;consumeSensitivityAnalysisResult;consumeSensitivityAnalysisStopped;consumeSensitivityAnalysisFailed;consumeShortCircuitAnalysisResult;consumeShortCircuitAnalysisStopped;consumeShortCircuitAnalysisFailed;consumeVoltageInitResult;consumeVoltageInitStopped;consumeVoltageInitFailed;consumeLoadFlowResult;consumeLoadFlowStopped;consumeLoadFlowFailed
```

Comment:
why prefix "SensitivityAnalysis", and not directly consumeNonEvacuatedEnergyResult as it's done elsewhere ?

Reply:
Renamed